aws-operator-user/iam-policy.json

```diff
@@ -44,17 +40,13 @@
                 "iam:RemoveRoleFromInstanceProfile",
                 "iam:UpdateAssumeRolePolicy",
                 "iam:UpdateRoleDescription",
-                "kms:*",
                 "logs:*",
                 "route53:*",
-                "route53domains:*",
```

Comment:
is this just some old naming or what is the scope?

Reply:
The scope is different from Route53, I checked aws-operator and so far we didn't use one of those calls:

```
AcceptDomainTransferFromAnotherAwsAccount
CancelDomainTransferToAnotherAwsAccount
CheckDomainAvailability
CheckDomainTransferability
DeleteTagsForDomain
DisableDomainAutoRenew
DisableDomainTransferLock
EnableDomainAutoRenew
EnableDomainTransferLock
GetContactReachabilityStatus
GetDomainDetail
GetDomainSuggestions
GetOperationDetail
ListDomains
ListOperations
ListTagsForDomain
RegisterDomain
RejectDomainTransferFromAnotherAwsAccount
RenewDomain
ResendContactReachabilityEmail
RetrieveDomainAuthCode
TransferDomain
TransferDomainToAnotherAwsAccount
UpdateDomainContact
UpdateDomainContactPrivacy
UpdateDomainNameservers
UpdateTagsForDomain
ViewBilling
```

Reply:
right its about issuing a DNS domain, make sense we don't need that